src/AAVE.jsx

```diff
@@ -112,10 +126,13 @@ if (
     .getNetwork()
     .then((data) => {
       const chainId = data?.chainId;
-      if (chainId && chainId === DEFAULT_CHAIN_ID) {
-        State.update({ chainId });
-      } else {
+      const config = getNetworkConfig(chainId);
+      if (!config) {
+        console.log(`Unsupport chain, chainId: ${chainId}`);
+        State.update({ isChainSupported: false });
         switchEthereumChain(DEFAULT_CHAIN_ID);
+      } else {
+        State.update({ chainId });
```

Comment:
```suggestion
        State.update({ chainId , isChainSupported: true });
```

It might be safer to also set `isChainSupported` if chain is supported.